exercises/solutions/solution_12.py

```diff
@@ -1,60 +1,92 @@
-from exercises.solutions.resources.resource_06_12 import KiwiPage
+from exercises.solutions.resources.resource_07_12 import KiwiPage
 
 
-# Calendar buttons (hit month and expect it in the date field)
-def test_hitting_calendar_buttons_is_reflected_by_date_field(page):
```

Comment:
I'd rather keep it as e.g. `task_13`, but it can be refined for clarity in the future and re-added afterwards.